frontend/src/metabase/api/channel.ts

```diff
@@ -0,0 +1,67 @@
+import type { NotificationChannel } from "metabase-types/api";
+
+import { Api } from "./api";
+import { idTag, invalidateTags, listTag, provideChannelListTags } from "./tags";
+
+const channelApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listChannels: builder.query<NotificationChannel[], void>({
+      query: () => `api/channel`,
```

Comment:
I realize that you probably didn't pick it, but `/api/channel` is not very descriptive